src/T2DMap.cpp

```diff
@@ -2781,7 +2781,7 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
                     getCenterSelection();
             }
 
-            if (!mpMap->mpRoomDB) {
+            if (mpMap->mpRoomDB->isEmpty()) {
```

Comment:
```suggestion
            if (!mpMap->mpRoomDB && mpMap->mpRoomDB->isEmpty()) {
```

Just for extra safety

Reply:
I'd say `||` not `&&`.

Reply:
Yes